src/config/dark-sites.config

```diff
@@ -235,6 +234,7 @@ destiny.gg
 destinytracker.com
 devanbuggay.com
 devart.withgoogle.com
+developer.mozilla.org
```

Comment:
MDN cannot be added they don't offer their dark mode by-default:
![image](https://user-images.githubusercontent.com/25481501/156345055-89dfdc75-3dde-4e4a-bfc8-c4364073f099.png)

Reply:
Ah. Makes sense.

Reply:
Hmmm... I get the dark mode by default if I have the user agent set to use a dark theme.  The little switch in the upper-right corner is flipped the other way for me.

Reply:
Then it detects the system's color scheme, but isnt qualified to be listed here.

Reply:
Ah, so do we only list sites that are *always* dark and not ones that auto-detect?

It will be nice to not have DR overriding dark themed sites that are dark because they auto-detected a dark themed user agent.

Reply:
I'm against adding more lists, those lists have become a nightmare and PR's like: https://github.com/darkreader/darkreader/pull/7995 should be focused on instead.

Reply:
I agree completely.  Love that idea.